entrypoint.sh

```diff
@@ -1313,6 +1313,17 @@ function print_loaded_drv_ver_str() {
     fi
 }
 
+function update_ca_certificates() {
+    timestamp_print "Updating system CA certificates (Ubuntu)..."
+
+    if command -v update-ca-certificates >/dev/null 2>&1; then
+        exec_cmd "update-ca-certificates"
```

Comment:
Maybe add `|| true` to not fail the whole process?

` exec_cmd "update-ca-certificates || true"`